tests/Functional/FormBuilderTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Symfony\Cmf\Component\ContentType\Tests\Functional;
+
+use Symfony\Cmf\Component\ContentType\Tests\Functional\Model\Article;
+
+class FormBuilderTest extends BaseTestCase
+{
+    public function testFormBuilder()
+    {
+        $builder = $this->getContainer([
+            'mapping' => [
+                Article::class => [
+                    'properties' => [
+                        'title' => [
+                            'type' => 'text',
+                        ],
+                    ],
+                ],
+            ],
+        ])->get('cmf_content_type.form_builder');
+
+        $article = new Article();
+        $builder = $builder->buildFormForContent($article);
+        $form = $builder->getForm();
```

Comment:
Not sure if the form builder should return the form or the builder. Returning the builder would be the more flexible approach I guess.

Should also rename that method.